gpu-operator/release-notes.rst

```diff
@@ -57,7 +62,7 @@ Known Issues
   GPU workloads cannot be scheduled on nodes that have a mix of MIG slices and full GPUs. 
   This manifests as GPU pods getting stuck indefinitely in the ``Pending`` state. 
   NVIDIA recommends that you downgrade the driver to version 570.86.15 to work around this issue.
-  For more detailed information, see GitHub issue #1361 <https://github.com/NVIDIA/gpu-operator/issue/1361>__.
+  For more detailed information, see GitHub issue https://github.com/NVIDIA/gpu-operator/issue/1361.
```

Comment:
This "Known issue" is now fixed in `570.172.08`. Let's update this section accordingly